lib/broadway/message.ex

```diff
@@ -57,10 +57,10 @@ defmodule Broadway.Message do
   end
 
   @doc """
-  Defines the batch key within a batcher for the message.
+  Defines the message batch key.
 
-  Inside each batcher, we attempt to build `batch_size`
-  within `batch_timeout` for each `batch_key`.
+  Batcher functions then attempt to create batches with the same `batch_key`,
+  of size `batch_size` within period `batch_timeout`.
```

Comment:
Not sure about this one. I think the old one tries to describe better what really happens. We need to make it clear that this key does not define the batcher. It defines how the batches are created inside each batcher.

Reply:
I guess it's equally clear and/or awkward either way. So keep the old one.